### DISCUSSION TITLE
Core_Encounter.class - будем ли профилировать?

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#256] [Клиническая РГ]
Encounter.class имеет привязку preferred к следующему [списку](https://terminology.hl7.org/5.1.0/ValueSet-encounter-class.html): 
<img width="1103" alt="image" src="https://github.com/fhir-ru/core/assets/62525329/d0634b0f-8fd3-4150-9ff9-548ff22846c3">

В то же время, в российском законодательстве (ФЗ-323, статья 32) есть следующие понятия, возможно связанные с классификацией случаев обслуживания:

- Виды медицинской помощи
-- первичная медико-санитарная помощь;
-- специализированная, в том числе высокотехнологичная, медицинская помощь;
-- скорая, в том числе скорая специализированная, медицинская помощь;
-- паллиативная медицинская помощь.
- Условия оказания помощи
-- вне медицинской организации
-- амбулаторно
-- в дневном стационаре
-- в круглосуточном стационаре
- Формы оказания помощи
-- экстренная
-- неотложная
-- плановая

Вопрос: будем ли мы профилировать encounter.class или писать рекомендации по его заполнению?

**Comment by Vrodvi** [comment]
Атрибут class имеет множественную кардинальность [0..*]:
![image](https://github.com/fhir-ru/core/assets/92928137/305b2a9a-7c11-44a1-a48e-059cd9187a0c)
, а сила привязки набора значений (preffered) позволяет использовать любой другой набор значений.

В связи с тем, что РФ fhir-ресурсы, скорее всего не будут участвовать ни в каком международном обмене информацией, то рекомендованный HL7 набор значенй использовать особого смысла нет.

Если имеется рускоязычный набор значений для этих целей, то он должен быть добавлен в ФНСИ. Такой справочник может быть рекомендован к использованию как один из возможных. Далее уже конкретные реализации под конкретный сценарий использования (в стационаре, амбулаторная помощь и прочие) смогут запрофилировать более жёстко.

Похоже такой справочник уже есть - https://nsi.rosminzdrav.ru/dictionaries/1.2.643.5.1.13.13.99.2.322/passport/3.1
по крайней мере он основан как раз на указанном 323-ФЗ
![image](https://github.com/fhir-ru/core/assets/92928137/d29483b5-3521-415c-96cb-0fe5d6036ad2)
